mat64/symmetric_test.go

```diff
@@ -266,8 +268,9 @@ func (s *S) TestRankTwo(c *check.C) {
 		s.RankTwo(a, alpha, NewVector(len(x), x), NewVector(len(y), y))
 		for i := 0; i < n; i++ {
 			for j := i; j < n; j++ {
-				v := m.At(i, j)
-				c.Check(s.At(i, j), check.Equals, v)
+				if !floats.EqualWithinAbsOrRel(s.At(i, j), m.At(i, j), 1e-15, 1e-15) {
```

Comment:
It doesn't really mater, but I've usually been using 1e-14 unless there is a good reason to be higher.